app/resources/applications/ApplicationsController.jl

```diff
@@ -408,12 +418,41 @@ macro isonline(app)
   end
 end
 
+function move_to_folder(app, from_folder, to_folder)
+  app_path = joinpath(from_folder, app.name)
+
+  if !isdir(app_path)
+    @error "File '$app' is not a directory."
+  end
+
+  if(isdir(joinpath(to_folder, app.name)))
+    timestamp = now() |> (dt -> trunc(dt, Minute)) |> (dt -> Dates.format(dt, "yyyy-mm-ddTHH:MM")) |> (s -> replace(s, r"[-:]" => ""))
+    app_new_name = app.name * timestamp
```

Comment:
Here we should have a separator between app name and timestamp, like `__` so we can split/undo the change.